doc/rst/source/supplements/seis/coupe.rst

```diff
@@ -136,6 +137,27 @@ Optional Arguments
     Give a CPT and let compressive part color be
     determined by the z-value in the third column.
 
+.. _-D:
+
+**-D**\ [**+c**][**+g**\ [*fill*]][**+o**\ *dx*\ [/*dy*]][**+p**\ *pen*][**+s**\ [*symbol*]\ *size*] ]
+    Offsets projected focal mechanisms to the alternate distance, depth given
+    in the last two columns of the input file before the (optional) text
+    string. Instead, if alternate geographical coordinates are given in the two columns then use **+c** to first
+    convert them to *distance, depth* coordinates in the crossectional plane, and after projection to
+    plot locations you can further adjust them via a fixed plot offset **+o**\ *dx/dy* .
+    Alternatively, use **+o** to interpret the contents of the two
+    columns as plot offsets instead, or append fixed offset *dx/dy* for all adjusted events plot locations.
+    We will draw a line connecting the original and relocated
+    beachball positions and optionally place a small symbol [circle] at the original
+    location.  Use **+s**\ *size* to set the diameter of the symbol [no symbol].
+    Change from circle to any of the standard geometric symbols in :doc:`plot </plot>`,
+    i.e., **a**\|\ **c**\|\ **d**\|\ **g**\|\ **h**\|\ **i**\|\ **n**\|\ **p**\|\ **s**\|\ **t**\|\ **x** [**c**].
+    The symbol will be filled with the beachball color, but a fixed color (**+g**\ *fill*)
+    or no fill (**+g**) can be selected.
+    The line pen defaults to that given via |-W| but can be overridden
+    by using **+p**\ *pen* [0.25p]. **Note**: If the cross symbol (**x**) is selected there is no fill since it is
+    just two stroked lines. Use |-W| to set line thickness and append **+c** to color the crosses as other symbols.
```

Comment:
>  **Note**: If the cross symbol (**x**) is selected there is no fill since it is
    just two stroked lines. Use |-W| to set line thickness and append **+c** to color the crosses as other symbols.

This sentence is no longer valid after PR https://github.com/GenericMappingTools/gmt/pull/7789 is merged, right?